src/Avalonia.X11/X11TrayIconImpl.cs

```diff
@@ -0,0 +1,369 @@
+﻿using System;
+using System.Diagnostics;
+using System.Reactive.Disposables;
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+using Avalonia.Controls.Platform;
+using Avalonia.FreeDesktop;
+using Avalonia.Logging;
+using Avalonia.Platform;
+using Tmds.DBus;
+
+[assembly: InternalsVisibleTo(Connection.DynamicAssemblyName)]
+
+namespace Avalonia.X11
+{
+    internal class X11TrayIconImpl : ITrayIconImpl
+    {
+        private static int s_trayIconInstanceId;
```

Comment:
Again ```s``` prefix.As I understand it is being used to explicitly say that you are calling static variable, but as I understand this prefix is not required by repo code style guidelines(if we even have them).If it is required by repo code style guidelines you are not consistant in usage of this prefix, I can see few places where you have static variable not marked with ```s_``` prefix.